examples/compare_connectivity_over_time_over_trial.py

```diff
@@ -0,0 +1,392 @@
+# return an instance of :class:`mne_connectivity.SpectroTemporalConnectivity`
+# and ``mode="fourier"`` or ``mode="multitaper"`` returns an instance of 
+# :class:`mne_connectivity.SpectralConnectivity`, which does not have
+# single timepoint resolution.
+
+con_epochs_array = np.mean(con_epochs_array, axis=4) # average over timepoints
+
+# In this example, we will just show alpha
+foi = list(Freq_Bands.keys()).index("alpha") # frequency of interest
+
+fig, ax = plt.subplots(1,n_con_methods, figsize=(6*n_con_methods,6))
+for c in range(n_con_methods):
+    ax[c].imshow(con_epochs_array[c,:,:,foi], cmap="binary", vmin=0, vmax=1)
+    ax[c].set_title(connectivity_methods[c])
+    print(f"Connectivity method: {connectivity_methods[c]}\n"+
+          f"{con_epochs_array[c,:,:,foi]}")
+
+###############################################################################
+# We see that when using repeated trials without any noise, the phase coupling
+# between the three electrodes over trials are exactly 1.
+#
+# We will now compute connectivity over time.
+
+# Pre-allocatate memory for the connectiviy matrices
+con_time_array = np.zeros((n_con_methods, n_epochs, n_channels,
+                           n_channels, n_freq_bands))
+con_time_array[con_time_array==0] = np.nan # nan matrix as 0 is meaningful
+
+# Compute connectivity over time
+con_time = spectral_connectivity_time(data_epoch, freqs,
+                                      method=connectivity_methods,
+                                      sfreq=sfreq, mode="cwt_morlet", 
+                                      fmin=fmin, fmax=fmax,
+                                      faverage=True)
+
+# Get data as connectivity matrices
+for c in range(n_con_methods):
+    con_time_array[c] = con_time[c].get_data(output="dense")
+
+###############################################################################
+# Notice that the connectivity over time function by default gives connectivity
+# for each epoch. We will average over epochs to show similar matrices as
+# before, but it could also be done in the function itself by setting
+# ``average=True``.
+
+con_time_array = np.mean(con_time_array, axis=1) # average over epochs
+foi = list(Freq_Bands.keys()).index("alpha") # frequency of interest
+
+fig, ax = plt.subplots(1,n_con_methods, figsize=(6*n_con_methods,6))
+for c in range(n_con_methods):
+    ax[c].imshow(con_time_array[c,:,:,foi], cmap="binary", vmin=0, vmax=1)
+    ax[c].set_title(connectivity_methods[c])
+    print(f"Connectivity method: {connectivity_methods[c]}\n"+
+          f"{con_time_array[c,:,:,foi]}")
+
+###############################################################################
+# We see that the connectivity over time are not 1, since the signals were
+# randomly generated and therefore the phase differences between channels
+# are also random over time.
+#
+# **Case 2: 10 Hz sinus waves with different phases.**
```

Comment:
Perhaps break up the sectioning of "conclusion" of the previous case with the start of "Case 2"?